manual-check.js

```diff
@@ -4,13 +4,19 @@
 //
 
 import Spark, { getRetrievalUrl } from './lib/spark.js'
+import { getMinerPeerId as defaultGetMinerPeerId } from './lib/miner-info.js'
 
 // The task to check, replace with your own values
-const cid = 'bafybeiepi56qxfcwqgpstg25r6sonig7y3pzd37lwambzmlcmbnujjri4a'
-const minerId = 'f010479'
+const cid = 'bafkreih25dih6ug3xtj73vswccw423b56ilrwmnos4cbwhrceudopdp5sq'
+const minerId = 'f0frisbii'
+
+const getMinerPeerId = (minerId) =>
```

Comment:
I got confused shortly because I didn't realize `minerId` is shadowing the outside value. I _believe_ standard would forbid that. I know other linters do. Could we please find a different variable name?

Reply:
One solution would be to rename the above variables to

```js
const task = {
  cid: '...',
  minerId: '...'
}
```